Makefile

```diff
@@ -43,22 +43,26 @@ clean: check-go ## runs `go clean` and removes the bin/ dir
 	$(GOCLEAN) --modcache
 	rm -rf $(GOBIN)
 
-cover: check-go ## runs test suite with coverage profiling
+cover: check-go static ## runs test suite with coverage profiling
 	$(GOTEST) ./... -coverprofile=$(COVER_FILE)
 
 cover-html: cover ## generates test coverage html report
 	$(GOCMD) tool cover -html=$(COVER_FILE)
 
 install: check-go ## installs all dev and ci dependencies, but does not install golang
 	curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s -- -b $(GOPATH)/bin v1.19.0
+	go get github.com/rakyll/statik
```

Comment:
Why isn't this automatically installed like other deps?

Reply:
This is a tool to embed static files into the binary. This command is installing the `statik` CLI which is run under the `static` make target.

Reply:
why do we need this package?  can we server directly using staticServer := http.FileServer(http.Dir(filesDir)) ?

Reply:
So we can still deploy a static binary and not have to incorporate this content into each deployment. Excerpt from the statik repo:
> Is this a crazy idea? No, not necessarily. If you're building a tool that has a Web component, you typically want to serve some images, CSS and JavaScript. You like the comfort of distributing a single binary, so you don't want to mess with deploying them elsewhere. If your static files are not large in size and will be browsed by a few people, statik is a solution you are looking for.